server/users/userController.js

```diff
@@ -66,7 +66,7 @@ userController.resetWithData = function() {
       email: null
     }).save();
   }).then(function() {
-    User.forge({
+    return User.forge({
```

Comment:
@alanzfu They're Promises/async functions, so they need to return in order for everything to be created in the same order consistently every time.